products/gitlab.md

```diff
@@ -21,8 +21,16 @@ auto:
 
 # EOL of R = releaseDate(R+3)
 # Support of R = releaseDate(R+1)
-# This is quite predictable since releases are monthly (usually 22nd of every month).
+# This is quite predictable since releases are monthly on third Thursday of every month (https://about.gitlab.com/blog/2023/09/18/gitlab-release-date-change/).
```

Comment:
The product description must be updated accordingly.

Reply:
We have the release cadence not yet in the description. It's a good idea to have it also on the website. Will add it...